integrated_channels/api/v1/moodle/serializers.py

```diff
@@ -12,8 +14,12 @@ class Meta:
             'moodle_base_url',
             'service_short_name',
             'category_id',
-            'username',
-            'password',
-            'token',
+            'encrypted_username',
```

Comment:
Do we currently have configs using decrypted data?

Reply:
@zamanafzal the goal was to only provide encrypted values other than where its decrypted value is required on our end i.e, while making requests using API client